src/Courses/CourseDetailsTabs/GradeTable.tsx

```diff
@@ -1,31 +1,64 @@
-import React from 'react';
+/* eslint-disable react/display-name */
+import React, {forwardRef} from 'react';
 import { Table } from 'react-bootstrap';
 import _ from 'lodash';
+import MaterialTable from 'material-table';
+import { Clear, SaveAlt, FilterList, FirstPage, LastPage, ChevronRight, ChevronLeft, Search, ArrowDownward } from "@material-ui/icons";
 
 interface GradeTableProps {
+    courseName: string;
     grades: Array<any>;
     onRowClick?: (id: number) => void;
 }
+const icons = {
```

Comment:
Should we move this somewhere global so we can reuse the icons?

Reply:
I'll resolve this in a new PR when I add the other Material UI component.